schema-nft-marketplace.graphql

```diff
@@ -0,0 +1,109 @@
+# Subgraph Schema: NFT Marketplace
+# Version: 0.0.1
+
+enum Network {
+  ARBITRUM_ONE
+  ARWEAVE_MAINNET
+  AURORA
+  AVALANCHE
+  BOBA
+  BSC # aka BNB Chain
+  CELO
+  COSMOS
+  CRONOS
+  MAINNET # Ethereum Mainnet
+  FANTOM
+  FUSE
+  HARMONY
+  JUNO
+  MOONBEAM
+  MOONRIVER
+  NEAR_MAINNET
+  OPTIMISM
+  OSMOSIS
+  MATIC # aka Polygon
+  XDAI # aka Gnosis Chain
+}
+
+enum NftStandard {
+    ERC721
+    ERC721A
+    ERC1155
+}
+
+enum SaleStrategy {
+    " Strategy that executes an order at a fixed price that can be taken either by a bid or an ask. "
+    StandardSaleForFixedPrice
```

Comment:
maybe can just be named `StandardSale` since the fixed price is mentioned in the comment and the only sale strategy that doesn't have a fixed price is `DutchAuction` (if i'm not mistaken)